internal/app/init/internal/reg/reg.go

```diff
@@ -72,7 +73,19 @@ func (r *Registrator) Upgrade(ctx context.Context, in *proto.UpgradeRequest) (da
 		return data, err
 	}
 
+	// stop kubelet
+	if _, err = r.Stop(ctx, &proto.StopRequest{Id: "kubelet"}); err != nil {
```

Comment:
we could have stopped the service directly here as well? I mean we don't need to call API, as we're in `init`